homework/index.js

```diff
@@ -29,22 +29,47 @@
     return elem;
   }
 
+  function createLiElement(ul, element1, element2, options1, options2) {
+    const li = createAndAppend('li', ul, {class: 'list'});
+    return createAndAppend(element1, li, options1),
+      createAndAppend(element2, li, options2);
+  }
```

Comment:
+1 for creating a function to make your code DRY.

The `return` keyword is unneeded here. What do you expect this function to return, and which caller would be interested in the value this function returns?

You can remove the `return` keyword so that you function becomes like this:

```js
function createLiElement(ul, element1, element2, options1, options2) {
  const li = createAndAppend('li', ul, { class: 'list' });
  createAndAppend(element1, li, options1);
  createAndAppend(element2, li, options2);
}
```